quick_tour/the_big_picture.rst

```diff
@@ -155,11 +155,12 @@ because that will be explained in the next section)::
 
     use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
     use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+    use Symfony\Component\HttpFoundation\Request;
```

Comment:
I am not sure if it makes things easier to understand for newcomers when they see a `use` statement for a class that is not used anywhere. It *could* give them the feeling that simply adding the `use` statement triggers some magic. However, let's see what Ryan and Wouter think about it.